examples/save_class_to_file.py

```diff
@@ -0,0 +1,51 @@
+from utilities import ensure_no_files_overwritten
+
+import jsonpickle
+
+ensure_no_files_overwritten(
+    expected_contents='{"py/object": "__main__.Example", "data": {"BAR": 1, "foo": 0}}'
+)
+
+
+class Example:
+    def __init__(self):
+        self.data = {"foo": 0, "BAR": 1}
+
+    def get_foo(self):
+        return self.data["foo"]
+
+    def __eq__(self, other):
+        # ensure that jsonpickle preserves important stuff like data across encode/decoding
+        return self.data == other.data and self.get_foo() == other.get_foo()
+
+
+# instantiate the class
+ex = Example()
+# encode the class. this returns a string, like json.dumps
+encoded_instance = jsonpickle.encode(ex)
+assert (
+    encoded_instance
+    == '{"py/object": "__main__.Example", "data": {"BAR": 1, "foo": 0}}'
+)
+print(
+    f"jsonpickle successfully encoded the instance of the Example class! It looks like: {encoded_instance}"
+)
+
+
+with open("example.txt", "w+") as f:
+    f.write(encoded_instance)
+    print(
+        f"jsonpickle successfully wrote the instance of the Example class to example.txt!"
```

Comment:
We might be able to use a regular string rather than an f-string here.

Reply:
Opps yeah, I think I previously had something printed out but forgot to remove the f